src/chrome/content/rules/Taobao.com-falsemixed.xml

```diff
@@ -1,15 +1,9 @@
 <!--
-	For rules not causing false/broken or valid MCB, see Taobao.xml.
+	For rules not causing false/broken or valid MCB, see Taobao.com.xml.
 -->
-
 <ruleset name="Taobao.com (MCB)" platform="mixedcontent">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="support.taobao.com" />
```

Comment:
If we redirect https?://assets.taobaocdn.com/ to https://s.tbcdn.cn/ , we can enable support.taobao.com